src/pudl/io_managers.py

```diff
@@ -855,12 +855,13 @@ def load_input(self, context: InputContext) -> pd.DataFrame:
             context: dagster keyword that provides access output information like asset
                 name.
         """
-        # TODO (daz): this is hard-coded to FERC1, though this is nominally for all FERC datasets.
-        ferc1_settings = context.resources.dataset_settings.ferc1
+        ferc_settings = context.resources.dataset_settings[
+            self.db_name.replace("_xbrl", "")
+        ]
 
         table_name = get_table_name_from_context(context)
         # Remove preceeding asset name metadata
-        table_name = table_name.replace("raw_ferc1_xbrl__", "")
+        table_name = table_name.replace(f"raw_{self.db_name}__", "")
```

Comment:
This was an attempt to not hard-code FERC1 to this function, but it causes issues with the tests in `test/unit/io_manager`.

The `test_ferc_xbrl_sqlite_io_manager_dedupes` function sets the `db_name` to `test_db` which causes problems here. Need to figure out a good work-around. Not sure whether to fix the tests or fix this reference.

Reply:
this seems like a good change! imo we should edit the test to enable it to deal with this change